### PR TITLE
Feature model photo

### PR DESCRIPTION
**Pull request by bleachjade**
I add an ImageField to model DB so that you can upload a photo when you create an event, display the most 3-4 recent event images on the index page, and fix some index, event detail, and create an event page.

### CODE REVIEW

**Comment by BenZacs** [comment]
After pull the code and run server everything can work properly, photo can be added and show correctly. The style look good but it seem like photo is not fit in the block like in the screen shot that I uploaded so, you can fix it later.

image

**Comment by BenZacs** [comment]
After pull the code and run server everything can work properly, photo can be added and show correctly. The style look good but it seem like photo is not fit in the block so, you can fix it later.

**Comment by BenZacs** [comment]
After pull the code and run server everything can work properly, photo can be added and show correctly. The style look good but it seem like photo is not fit in the block so, you can fix it later.

File: Kvent/templates/kvent/event-detail.html [hunk 1]
```diff
@@ -17,11 +17,16 @@
     </nav>
     <div class="row">
         <hr class="line">
+        <div class="event-image">
+            <img src="{{ event.get_photo.url }}" alt="sad" width="300px" height="auto" />
```
Comment (forfeen) [inline Kvent/templates/kvent/event-detail.html:21]: what is **/** after height attribute?

File: Kvent/views.py [hunk 2]
```diff
@@ -52,19 +53,22 @@ def detail(request, event_id):
 
 def create_event(request):
     """ User creates the event """
-    form = EventForm(request.POST)
+    form = EventForm(request.POST, request.FILES)
     if request.method == 'POST' :
         if form.is_valid() :
+            photo = form.cleaned_data.get('photo') 
+            print(photo)
```
Comment (forfeen) [inline Kvent/views.py]: Why you print out the data ?

File: Kvent/views.py [hunk 3]
```diff
@@ -52,19 +53,22 @@ def detail(request, event_id):
...
             event_name = form.data.get('event_name')
+            print(event_name)
```
Comment (forfeen) [inline Kvent/views.py]: Remove _print(event_name)_

File: Kvent/templates/kvent/event-detail.html (hunk 1 above)
Comment (bleachjade) [reply Kvent/templates/kvent/event-detail.html:21]: I use "/" because I'm to lazy to write `</img>`

File: Kvent/views.py (hunk 3 above)
Comment (bleachjade) [reply Kvent/views.py]: I print to check the output data, I will delete these out.

**Comment by bleachjade** [comment]
I already fixed the photo style not fit in the event-box.

**Pull request merged**